src/components/ui/drawers/TaskDeletionDrawer.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Drawer,
+  DrawerContent,
+  DrawerFooter,
+  DrawerTitle,
+  DrawerDescription,
+  DrawerTrigger,
+} from '@/components/ui/drawer';
+import { useForm } from 'react-hook-form';
+import { Button } from '@/components/ui/button';
+
+interface ITaskDeletionDrawer {
+  title: string;
+  open: boolean;
+  setOpen: (value: boolean) => void;
+}
+
+/**
+ * A drawer component to delete a task
+ */
+export const TaskDeletionDrawer = ({
+  title,
+  open,
+  setOpen,
+}: ITaskDeletionDrawer) => {
+  const { handleSubmit } = useForm();
+
+  // TODO: Implement the delete click functionality
+  const onSubmit = () => {
+    setTimeout(() => {
+      setOpen(false);
+    }, 1000);
+  };
+
+  return (
+    <Drawer open={open} onOpenChange={setOpen}>
+      <DrawerTrigger />
+      <DrawerContent asChild>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <DrawerTitle>Delete</DrawerTitle>
+          <DrawerDescription>
+            <p className={'text-base mt-4'}>
+              You want to delete &quot;{title}&quot; ?
+            </p>
+          </DrawerDescription>
+          <DrawerFooter className={'flex justify-between'}>
+            <Button
+              type={'button'}
+              variant={'outline'}
+              onClick={() => setOpen(false)}
+              className={'flex-1'}
+            >
+              Cancel
+            </Button>
```

Comment:
IMO, it's better to use `<DrawerClose>` instead of `onClick={() => setOpen(false)}`.

▼shadcn Usage: https://ui.shadcn.com/docs/components/drawer
![Screenshot 2024-06-19 at 6 19 58 PM](https://github.com/Tascurator/tascurator-frontend/assets/132441773/7caa7afb-3811-4900-bf89-2e2a0ea94756)

▼[TaskCreationDrawer.tsx](https://github.com/Tascurator/tascurator-frontend/pull/137/files#diff-2a66389c799fa4471180b86eeb4ec9bf8ff16ed48661eceade113692959848ea)
> ConfirmTaskDrawer component
![Screenshot 2024-06-19 at 6 22 09 PM](https://github.com/Tascurator/tascurator-frontend/assets/132441773/c2762599-8f39-44c7-92f0-c8bf4a54ae1d)